release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -132,6 +132,33 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic Plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Administration* -> *Cluster Settings* page. You can also add different extension types to customize your plug-in. 
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dunamic-plug-ins_web_console[Dynamic plug-ins].
+
+[id="ocp-4-11-dark-mode"]
+==== Support for dark mode theme
+
+The {product-title} web console now supports the dark mode theme. On the *User Settings* page, select your preferred mode to view the web console in.
+
+[id="ocp-globally-installed-operators-display-operand-instances"]
+==== Display operand instances for all managed namespaces on the *Operator Details* page
+
+When viewing the *Installed Operators* page set to `All projects` and then selecting an Operator that is available in the globally installed `All namespaces` to view the details of that Operator in installed namespace.
+
+[id="ocp-4-11-conditional-updates"]
+==== Conditional updates
+
+With this update, you can "include supported not recommended versions" for conditional updates to populate the list. If you select "supported, but not recommended" an alert will appear and display the potential issues. You can also view the "supported, but not recommended" versions" from the *Administration* -> *Cluster Settings* page.
+
+[id="ocp-4-11-pod-disruption-budgets"]
+==== Pod disruption budgets (PDBs)
+
+This update adds support for pod disruption budgets (PDBs) to the {product-title} web console. From *Workloads* -> *PodDisruptionBudget* you can create PDBs for pod resources. You can select `poddisruptionbudget` and change the availability requirement.
```

Comment:
@opayne1 I think it should be : *PodDisruptionBudgets*  as displayed on the web console UI side nav
`poddisruptionbudget`  should be `PodDisruptionBudget` 
cc: @jhadvig